modules/syslogformat/tests/corpus/pattern

```diff
@@ -0,0 +1 @@
+<45>1 2015-08-12T13:23:12+02:00 zts-win101 syslog-ng 2876 2 [meta sequenceId=\"99\"] syslog-ng Agent shutting down; version='5.5.0alpha
```

Comment:
Why did you escape 99? Copy paste error?

Reply:
I mean why the quotes are escaped around 99.

Reply:
yeah, just copy paste error